diff --git a/dist/chunk-RROAC55K.js b/dist/chunk-RROAC55K.js
index 7d6a36401906b619c1ef39c36e4f38f83b740c51..5b3a4d43b465158b5f8d2ca414ec0809f92eced1 100644
--- a/dist/chunk-RROAC55K.js
+++ b/dist/chunk-RROAC55K.js
@@ -4776,8 +4776,8 @@ function ListPage(props) {
   }
   const rows = React93.useMemo(() => {
     let result;
-    if (useClientSideFiltering && listLogic.sortedItems) {
-      result = listLogic.sortedItems;
+    if (useClientSideFiltering && listLogic.paginatedItems) {
+      result = listLogic.paginatedItems;
     } else if (directItems !== void 0) {
       result = directItems;
     } else if (!query?.data) {
@@ -4790,7 +4790,7 @@ function ListPage(props) {
       return sorted;
     }
     return result;
-  }, [useClientSideFiltering, listLogic.sortedItems, directItems, query?.data, getRows, isCardsView, resolvedCardSort, cardSortValue, isServerSideSort]);
+  }, [useClientSideFiltering, listLogic.paginatedItems, directItems, query?.data, getRows, isCardsView, resolvedCardSort, cardSortValue, isServerSideSort]);
   const rawTotal = React93.useMemo(() => {
     if (useClientSideFiltering) {
       return listLogic.filteredCount;
